components/ProjectCard.tsx

```diff
@@ -22,10 +21,26 @@ const ProjectCard: React.FC<ProjectCardProps> = ({
     personalTwitter,
     nym,
     zaprite,
+    tags,
   } = project
 
+  let cardStyle
+  if (tags.includes('Nostr')) {
+    cardStyle =
+      'h-full space-y-4 rounded-xl border-b-4 border-purple-600 bg-stone-100 dark:border-stone-800 dark:bg-stone-900'
+  } else if (tags.includes('Lightning')) {
+    cardStyle =
+      'h-full space-y-4 rounded-xl border-b-4 border-yellow-300 bg-stone-100 dark:border-stone-800 dark:bg-stone-900'
+  } else if (tags.includes('Bitcoin')) {
+    cardStyle =
+      'h-full space-y-4 rounded-xl border-b-4 border-orange-400 bg-stone-100 dark:border-stone-800 dark:bg-stone-900'
+  } else {
+    cardStyle =
+      'h-full space-y-4 rounded-xl border-b-4 border-stone-100 bg-stone-100 dark:border-stone-800 dark:bg-stone-900'
```

Comment:
NIT: lots of repeated code, the only thing that changes is the color.

Reply:
Yes I noticed that (obviously, as I wrote it). Tailwind seems not to like dynamic colors, at least on my machine. Will look into how to clean this up, perhaps as part of a larger refactoring effort.